COC_PyBay.md

```diff
@@ -0,0 +1,76 @@
+
+###Sponsors, affiliates, and exhibitors
+
+When you sponsor a PyBay event, we welcome you as a member of our community, and we expect you to be respectful to the community you operate within.
+
+All exhibitors in the expo hall, sponsor or vendor booths, or similar activities are also subject to the code of conduct. In particular, exhibitors should not use sexualized images, activities, or other material. Booth staff (including volunteers) must not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
+
+In addition, sponsors and affiliates of conference, meetups, and online activities should not employ aggressive recruiting techniques, invasive marketing behavior, or similar actions towards community members. In case of violations, sponsors might be sanctioned and expelled from the event or activity with no return of the sponsorship contribution.
+
+###Weapons Policy
+
+No weapons are allowed at conference venues, including but not limited to explosives (including fireworks), guns, and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately, and will only be allowed to return without the weapon.
+
+Attendees are further expected to comply with all state and local laws on this matter.
+
+###Young attendees
+
+Attendees of all ages are welcome at PyBay! Attendees 15 years of age and under are welcome to attend with a legal guardian. Both people should be registered for the conference and have badges. The legal guardian should be with the child at all times and the child should never be left alone at the conference.
+
+##What to do in case of violations
+
+If you believe that someone is violating the Code of Conduct during one of our events, please contact the CoC team immediately by emailing us at conduct@pybay.com.
+
+To help us respond in the best way to the situation, please follow the Code of Conduct Reporting. The CoC representatives employ the Code of Conduct Response to handle reports.
+
+If you believe a member of staff is violating our Code of Conduct, please reach out to one of the conference chairs, Slava (slavabal@gmail.com) or Hemanti (hemantisd@gmail.com).
+
+All reports will be kept confidential where legally possible. In some cases a public statement might be required (for example in a CoC transparency report following conferences), but these reports are anonymized and do not include any personally identifying information. In the event of involvement with law enforcement, we will comply with their requests as legally required.
+Thanks
+
+This code of conduct is largely based on the Write The Docs Code of Conduct, which in turn is based on the Django Project Code of Conduct and the original text of the Speak Up! project, inspired in its turn by the Fedora Project, as well as the Python Mentorship Project and many others.
```

Comment:
Can we also credit PyCascades, the immediate source of this?

Reply:
agreed.